prompting-101/improving-chatGPTs-responses.md

```diff
@@ -456,6 +456,103 @@ Let's take a look at one more example prompt for inspiration, one that puts all
+   - Add a response to the question "Anything else ChatGPT should know about you?"
+      - A response could look like: "It is very important to me that I come up with code solutions without looking at examples first. I want responses to my questions to provide overarching ideas without code examples that I can review so I can get more practice writing code by myself."
+
+![Custom Instructions Details Example](./assets/custom-instructions-without-code.png)  
+*Fig. Example of Custom Instructions for ChatGPT ([Full size image](assets/custom-instructions-without-code.png))*
+
+<details>
+   <summary>Expand this section to see an example prompt asking for help after providing Custom Instructions to not provide code in responses (from July 2025).</summary>
+
+> Please provide me a solution for the following problem: Given a string `s`, find the first non-repeating character in it and return its index. If it does not exist, return -1.
+
+> To solve this problem, here's a step-by-step approach in plain language, without code:
+> 
+> Count Occurrences:
+> Go through each character in the string and count how many times it appears. You can use a dictionary-like structure to map each character to its count.
+> 
+> Identify First Unique Character:
+> After counting, go through the string a second time—character by character—and check the count for each one.
+> 
+> Return the Index:
+> As soon as you find a character that appears only once, return its index.
+> 
+> If None Found:
+> If you finish checking and none of the characters are unique, return -1.
+> 
+> Let me know if you'd like a hint on how to structure the count or loop logic.
+</details> 
+
+**Reviewing AI responses**  
+If we do get code results (intentionally or unintentionally) in responses to our questions, we should still carefully review the response to make sure we understand not just the syntax of what is being suggested, but also the reason *why* each step is in a solution. 
+
+If there are areas of the AI response that are unclear, we should ask for further clarification and engage in a conversation to gain additional understanding before using any solution provided by an AI. While we could ask additional questions to the AI, it can also be very productive to engage with other Adies in #study-hall in Slack or with instructors and volunteers during office hours to learn different perspectives.
+
+Ideally, after reviewing the AI response, we should be able to write approximately the same approach on our own without referring back to the output from the AI, and be able to explain what the code does from both a syntax perspective, and a logcial perspective.
+
 ## Summary 
```

Comment:
I had been thinking that we could reiterate some of the learning strats in the summary, but reading through the way the existing summary is structured, it feels like we'd be shoehorning that material in, so I'm good leaving this as is.